text/0070-pd-record-historical-hot-region.md

```diff
@@ -0,0 +1,163 @@
+# Record history hotspot information in TiDB_HOT_REGIONS_HISTORY
+
+- RFC PR: https://github.com/tikv/rfcs/pull/0070
+- Tracking Issue: https://github.com/pingcap/tidb/issues/25281
+- Proposal Documention: https://docs.google.com/document/d/1KqjYBWhJ-YlupyDvBlObkCl-9P8WyUPL6VgYpO-LRiw/
+
+## Summary
+
+Create a new table `TIDB_HOT_REGIONS_HISTORY` in the `INFORMATION_SCHEMAT` database to retrieve history hotspot information stored by PD periodically.  
+
+## Motivation
+
+TiDB has a memory table `TIDB_HOT_REGIONS` that provides information about hotspot regions. 
+But it only shows the current hotspot information. This leads to the fact that when DBAs want to query historical hotspot details, 
+they have no way to find the corresponding hotspot information.
+
+## Background
+According to the [documentation](https://docs.pingcap.com/tidb/stable/information-schema-tidb-hot-regions) for the current `TIDB_HOT_REGIONS` table, 
+the table provides information about hotspot regions shown in the following table: 
```

Comment:
In Background, I think we should provide  what `TiDB_HOT_REGIONS_HISTORY` will look like and in which specific cases or problems, we can use `TiDB_HOT_REGIONS_HISTORY` to solve them and how.

Reply:
Add a example sql and result row in Background section.